.travis.yml

```diff
@@ -72,12 +82,27 @@ jobs:
             - python-pip
 
       install:
-        - cd $TRAVIS_BUILD_DIR/kuzzle-sdk-java
-        - gradle assemble
-        - cd $TRAVIS_BUILD_DIR/kuzzle-sdk-java
-        - gradle assemble
-      script:
-        - gradle check
+        provider: script
+        script:
+          - npm run doc-upload
+        skip_cleanup: true
+
+      after_deploy:
+        - npm run doc-cloudfront
+
+    - stage: Deployment
```

Comment:
Where does this stage come from?

Reply:
IDK

Reply:
@jenow Is it you?

Reply:
It has always been here https://github.com/kuzzleio/sdk-java/blob/master/.travis.yml#L97
Maybe see with @alexandrebouthinon?

Reply:
I just thought that @xbill82 add it.  
Are we gonna deploy the v3 of the SDK with this now @alexandrebouthinon ?